Running-Mastodon/Production-guide.md

```diff
@@ -437,7 +437,7 @@ User=mastodon
 WorkingDirectory=/home/mastodon/live
 Environment="RAILS_ENV=production"
 Environment="DB_POOL=5"
-ExecStart=/home/mastodon/.rbenv/shims/bundle exec sidekiq -c 5 -q default -q mailers -q pull -q push
+ExecStart=/home/mastodon/.rbenv/shims/bundle exec sidekiq -c 5 -q default -q push -q pull -q mailers
```

Comment:
`-q default,6 -q push,4 -q mailers,2 -q pull`

Reply:
Oops, I actually assumed `mailers` was only for the digest worker, but it's also implicitly used by `ActionMailer`. So yeah, it should probably be of higher priority than `pull`.
Not sure about the weights, it's a much bigger change than the reordering (since federation lag will now have a much bigger impact on local-only jobs).